Source/relay/TourGuide/Items of the Month/2021/Underground Fireworks Shop.ash

```diff
@@ -53,7 +53,7 @@ void IOTMUndergroundFireworksShopGenerateResource(ChecklistEntry [int] resource_
 		{
 			string [int] description;
 			description.listAppend("Can buy one of the following (500 meat):");
-			description.listAppend("Fedora-mounted mountain: +20 ML hat");
+			description.listAppend("Fedora-mounted fountain: +20 ML hat");
 			description.listAppend("Sombrero-mounted sparkler: +5% combat hat");
 			description.listAppend("Porkpie-mounted popper: -5% combat hat");
 			resource_entries.listAppend(ChecklistEntryMake("__item fedora-mounted mountain", "clan_viplounge.php?action=fwshop&whichfloor=2", ChecklistSubentryMake("Dangerous hats", description), 8).ChecklistEntrySetIDTag("Clan fireworks hat resource"));
```

Comment:
Gotta change this one too for the image to be right

Reply:
yep, gotta do that. will do after today's run